### ISSUE TITLE
Login help request, need to recover login info

### ISSUE THREAD

**Issue by jaxford** [blogs-perl-org/blogs.perl.org#303]
It has been several years since I've logged into blogs.perl.org and am now unable to do so. I tried what I thought was my username (hackster) but the passwords I tried failed. I asked Dave Cross for help and he told me j0e@yardbirdfanclub.org is the email associated with my account. I requested a password to be sent to j0e@yardbirdfanclub.org but none was received. I tried to login with j0e@yardbirdfanclub.org with what I thought might be a valid password and received the following message:

Invalid login. Please note that Movable Type wants you to log in with your username, not your email address.

Any suggestions?

**Comment by davorg** [comment]
Your username is j0e. If try that, if I was you.

**Comment by davorg** [comment]
And it's not fair to say you got no password reset email. It bounced, I got the bounce message and I forwarded it to you.

There are enough real problems with blogs.perl. org without people making more up ;-)

**Comment by jaxford** [comment]
Dave, I had setup email forwarding for j0e@yardbirdfanclub.org and tested it immediately before requesting a password from blogs.perl.org, and it forwarded correctly. It's only after I requested a password reset from blogs.perl.org that I discovered forwarding was no longer working. I wouldn't have submitted this issue had I known j0e@yardbirdfanclub.org was no longer being forwarded correctly, and I wouldn't have submitted this issue had I read the bounce you forwarded to me. I sincerely apologize, this was not intentional.

I since have discovered that email forwarding works intermittently for the first 30 minutes or so after being enabled, at least that is what my domain provider told me. That would explain why my first test of j0e@yardbirdfanclub.org worked, but subsequent mail forwarding failed. Some 60 or more minutes later forwarding is working consistently and I successfully changed my blogs.perl.org password and logged in.

Again, I am very sorry for the confusion, it was not intentional.

Thank you for your help Dave.
